tuna/miopen/db/miopen_tables.py

```diff
@@ -351,18 +322,8 @@ class FinStep(enum.Enum):
   miopen_perf_eval = 9
 
 
-class JobMixin():
-  """Represents Mixin class for job tables"""
-
-  @declared_attr
-  def session(self):
-    """session key"""
-    return Column(Integer, ForeignKey("session.id"), nullable=False)
-
-  reason = Column(String(length=60), nullable=False, server_default="")
-  state = Column(Enum(JobEnum), nullable=False, server_default="new")
-  retries = Column(Integer, nullable=False, server_default="0")
-  result = Column(Text, nullable=True)
+class MIOpenJobMixin(JobMixin):
```

Comment:
```suggestion
class MIOpenJobMixin(BASE, JobMixin):
```

all tables defined in tuna need to inherit the BASE class, that is our standard (unless they are defined as a Mixin)